src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -601,8 +601,19 @@ protected override void HandleLaunchRequestAsync(IRequestResponder<LaunchArgumen
             int hr;
             DateTime launchStartTime = DateTime.Now;
 
-            string mimode = responder.Arguments.ConfigurationProperties.GetValueAsString("miMode");
-            string program = responder.Arguments.ConfigurationProperties.GetValueAsString("program")?.Trim();
+            Dictionary<string, JToken> caseInsensitiveLaunchConfiguration;
+            try
+            {
+                caseInsensitiveLaunchConfiguration = new Dictionary<string, JToken>(responder.Arguments.ConfigurationProperties, StringComparer.OrdinalIgnoreCase);
+            } 
+            catch (ArgumentException e)
+            {
+                responder.SetError(CreateProtocolExceptionAndLogTelemetry(telemetryEventName, 1013, "Duplicate keys found in launch configuration."));
```

Comment:
The format is `<type>: <message>` so `launch: duplicate keys found in configuration`